### DISCUSSION TITLE
What part(s) of  Instructor Training Checkout is/are the most challenging for the trainees, the community, or both? Why?

### DISCUSSION THREAD

**Discussion by klbarnes20** [carpentries/instructor-training#1436] [checkout-feedback]
### Background
When someone wants to be a certified Carpentries Instructor, they first take Instructor Training. After that, they must complete checkout to be certified.

**[Checkout](https://carpentries.github.io/instructor-training/checkout/) consists of three steps:**

- Attend a community discussion

- Make a contribution to a Carpentries lesson (including those in the incubator) or other resources

- Do a 5 minute teaching demo 

The Core Team is working on a project that is the first systematic review and opportunity to improve the checkout process since 2018. **We want to hear from you!**

**Comment by sarasrking** [comment]
I can't really speak for that many people but I do know that a lot of trainees get super nervous about the 5 min demo. That's related to performance anxiety usually, not even lack of experience just having to do it with strangers knowing that they are going to give you direct feedback. But I know that's also the best part of the checkout so I wouldn't change it!

**Reply by santoshkarthikeyan** [reply]
It would be good to take offline teaching demo review and feedback as well through recorded demo considering the time zone differences.
